cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -194,8 +195,11 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
             throw ProgramResult(1)
         }
 
-        if (scanResults.hasIssues) {
-            println("The scan result contains issues.")
+        val hasIssues = scanResults.scanResults.any { (_, results) ->
+            results.any { it.summary.issues.any { issue -> issue.severity != Severity.HINT } }
+        }
+        if (hasIssues) {
+            println("The scan result contains non-hint issues.")
```

Comment:
While "non-hint" is correct and future-proof WRT additions to `Severity`, should we still use "errors or warnings" for user friendliness?

Reply:
You mean just `"The scan result contains errors or warnings."` ?

Reply:
Yes.